### ISSUE TITLE
Optional tags throw error

### ISSUE THREAD

**Issue by irisSchaffer** [nfl/react-helmet#255]
Version: 5.0.0
React Version: 15.4.2

I'm experiencing problems with rendering optional tags as so:

```
{track.getIn(['meta', 'description']) && (
    <meta name="description" content={track.getIn(['meta', 'description'])} />
)}
```

Gives me a `TypeError: Cannot read property 'props' of null`. I believe it is this line that is throwing the error: https://github.com/nfl/react-helmet/blob/master/src/Helmet.js#L201

If I, however, go with something like
```
{track.getIn(['meta', 'description']) && (
    <meta name="description" content={track.getIn(['meta', 'description'])} />
) || <meta />}
```

everything works fine and I get the correct description (which was set in the parent).

**Comment by doctyper** [comment]
Hm. We do test this specific scenario in our unit tests. Could you post a more reproducible example without the dependency on track.getIn?

**Comment by irisSchaffer** [comment]
Sure, of course, sorry! it resolves to `undefined` on the first render, so:

```
{undefined && (
    <meta name="description" content={undefined} />
)}
```

silly test case, but it seems to still receive some form of child, and therefore breaks :/
Mind you, I also have other children within Helmet:

```
<Helmet>
    <title>Test Title</title>

    {undefined && (
        <meta name="description" content={undefined} />
    )}
</Helmet>
```

**Comment by doctyper** [comment]
@irisSchaffer Fixed in `react-helmet@5.0.1`.
